parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -115,9 +122,12 @@ def __init__(self,
         self.block_id = block_id
 
         cores_on_node = multiprocessing.cpu_count()
+        available_mem_on_node = round(psutil.virtual_memory().available / (2**30), 1)
+
         self.max_workers = max_workers
         self.prefetch_capacity = prefetch_capacity
         self.worker_count = min(max_workers,
+                                math.floor(available_mem_on_node / mem_per_worker),
```

Comment:
I think it should check if `mem_per_worker` is not 0 or none?

Reply:
Fixed.